BUG: Default values file for spark-infrastucture-chart causes errors

## Description
The default `spark-infrastructure` chart issues warnings on installation.

## Steps to Reproduce
Clear, specific, and detailed steps taken to enable reproduction of the bug for investigation.
1. Create a project from the archetype
    ```sh
    mvn archetype:generate '-DarchetypeGroupId=com.boozallen.aissemble' \
                           '-DarchetypeArtifactId=foundation-archetype' \
                           '-DarchetypeVersion=1.13.0-SNAPSHOT' \
                           '-DgroupId=org.test' \
                           '-Dpackage=org.test' \
                           '-DprojectGitUrl=test.org/test.git' \
                           '-DprojectName=Test Project' \
                           '-DartifactId=bug-622' \
    && cd bug-622
    ```
2. Add an execution for the `spark-infrastructure-deploy-v2` Fermenter profile to _bug-622-deploy/pom.xml_
    ```xml
                        <execution>
                            <id>spark-infrastructure</id>
                            <phase>generate-sources</phase>
                            <goals>
                                <goal>generate-sources</goal>
                            </goals>
                            <configuration>
                                <basePackage>org.test</basePackage>
                                <profile>spark-infrastructure-deploy-v2</profile>
                                <propertyVariables>
                                    <appName>spark-infrastructure</appName>
                                </propertyVariables>
                            </configuration>
                        </execution>
    ```
3. Generate the chart
    ```sh
    mvn clean install
    ```
4. Deploy the resulting Helm chart
    ```sh
    helm install spark-infrastructure bug-622-deploy/src/main/resources/apps/spark-infrastructure
    ```

### Expected Behavior
The Helm chart installs without error.

### Actual Behavior
Helm issues a warning saying the property `aissemble-hive-metastore-service-chart.configMap.metastoreServiceConfig.properties` cannot be set because the value is a list and it's overriding a table.

## Additional Context
aiSSEMBLE version: 1.12.1
Error output:
```sh
coalesce.go:298: warning: cannot overwrite table with non table for spark-infrastructure.aissemble-hive-metastore-service-chart.configMap.metastoreServiceConfig.properties (map[])
coalesce.go:298: warning: cannot overwrite table with non table for spark-infrastructure.aissemble-hive-metastore-service-chart.configMap.metastoreServiceConfig.properties (map[])
coalesce.go:298: warning: cannot overwrite table with non table for spark-infrastructure.aissemble-hive-metastore-service-chart.configMap.metastoreServiceConfig.properties (map[])
NAME: spark-infrastructure
LAST DEPLOYED: Thu Mar 20 08:55:32 2025
NAMESPACE: default
STATUS: deployed
REVISION: 1
```

## Test Steps
1. Follow the steps to reproduce and ensure no warnings are printed
    1. **OTS Only**: Before deploying the helm chart (step 4), update _bug-622-deploy/src/main/resources/apps/spark-infrastructure/Chart.yaml_ to set the **repository** for `aissemble-hive-service-chart` to the local path, and the **version** to 1.0.0
    2. Update the dependencies with `helm dep up bug-622-deploy/src/main/resources/apps/spark-infrastructure`
3. Uninstall the chart: `helm uninstall spark-infrastructure`

--- a/extensions/extensions-helm/extensions-helm-spark-infrastructure/aissemble-hive-metastore-service-chart/values.yaml
+++ b/extensions/extensions-helm/extensions-helm-spark-infrastructure/aissemble-hive-metastore-service-chart/values.yaml
@@ -70,4 +70,4 @@ configMap:
       - name: javax.jdo.option.ConnectionURL
         value: jdbc:mysql://hive-metastore-db:3306/metastore?createDatabaseIfNotExist=true&amp;allowPublicKeyRetrieval=true&amp;useSSL=false
         description: JDBC connect string for a JDBC metastore
-    properties: {}
+    properties: []

